report-viewer/src/components/ClusterRadarChart.vue

```diff
@@ -5,98 +5,125 @@
 <template>
   <div class="wrapper">
     <select v-model="selectedMember">
-      <option v-for="(member, index) in Object.keys(cluster.members)" :key="index">{{ member }}</option>
+      <option v-for="(member, index) in cluster.members.keys()" :key="index">
+        {{ member }}
+      </option>
     </select>
-    <RadarChart :chartData="chartData" :options="options" class="chart"></RadarChart>
+    <RadarChart
+      :chartData="chartData"
+      :options="options"
+      class="chart"
+    ></RadarChart>
   </div>
 </template>
 
-<script>
-import {defineComponent, ref, watch} from "vue";
-import {RadarChart} from "vue-chart-3"
-import {Chart, registerables} from 'chart.js';
-import ChartDataLabels from 'chartjs-plugin-datalabels';
+<script lang="ts">
+import { defineComponent, PropType, Ref, ref, watch } from "vue";
+import { RadarChart } from "vue-chart-3";
+import { Chart, ChartData, registerables } from "chart.js";
+import ChartDataLabels from "chartjs-plugin-datalabels";
+import { ClusterListElement } from "@/model/ClusterListElement";
 
 Chart.register(...registerables);
 Chart.register(ChartDataLabels);
 
 export default defineComponent({
   name: "ClusterRadarChart",
-  components: {RadarChart},
+  components: { RadarChart },
   props: {
-    cluster: {}
+    cluster: {
+      type: Object as PropType<ClusterListElement>,
+      required: true,
+    },
   },
   setup(props) {
-    const selectedMember = ref("")
+    const getIdOfFirstSubmission = () =>
+      props.cluster.members.keys().next().value;
```

Comment:
This cannot or rather should not happen:) But I would suggest showing a, for now, **very** simple ErrorMessage instead of the chart: <img width="553" alt="Screenshot 2022-06-30 at 15 29 38" src="https://user-images.githubusercontent.com/33717535/176689890-804b0197-36c7-46c2-bd1b-e6cb3c1a0d82.png">.
Once we reach the `Task Enhance logging and error reporting in both JSON generator and report viewer code` we could improve upon this error message.